source/whats-new.txt

```diff
@@ -22,6 +22,293 @@ What's New in 4.0
 
 New features of the 4.0 Node.js driver release include:
 
+TypeScript
+~~~~~~~~~~
+
+- We've migrated the driver to TypeScript. You can now harness the type
+  hinting and intellisense features in editors that support it to develop
+  your MongoDB applications. You do not have to adopt TypeScript to get
+  the benefits of this work.
```

Comment:
```suggestion
  your MongoDB applications. You can get the benefits of this work even 
  in pure JavaScript projects!
```
Just taking a positive tone, total nit. I think we have the same lingo and I'll update too

Reply:
Great suggestion, thanks!